ui/pages/Swap.tsx

```diff
@@ -635,6 +636,16 @@ export default function Swap(): ReactElement {
                 }}
                 label={t("swap.to")}
               />
+              {priceDetails === undefined &&
+              sellAmount &&
```

Comment:
Let's do it like this so it goes both ways 😄 
```suggestion
              (sellAmount || buyAmount) &&
```